Sources/BraveWallet/Settings/CustomNetworkDetailsView.swift

```diff
@@ -213,7 +214,11 @@ class CustomNetworkModel: ObservableObject, Identifiable {
     self.networkSymbol.input = network.symbol
     self.networkDecimals.input = String(network.decimals)
     if !network.rpcEndpoints.isEmpty {
-      self.rpcUrls = network.rpcEndpoints.compactMap { NetworkInputItem(input: $0.absoluteString) }
+      var result: [NetworkInputItem] = []
+      for (index, endpoint) in network.rpcEndpoints.enumerated() {
+        result.append(NetworkInputItem(input: endpoint.absoluteString, isSelected: index == network.activeRpcEndpointIndex))
```

Comment:
nit: indentation (can we split this initializer to multiple lines?)

Reply:
updated in [9af3a40](https://github.com/brave/brave-ios/pull/7628/commits/9af3a4038dda48b591f837e187d77756165f85c2)